gdal/doc/source/tutorials/geotransforms_tut.rst - new tutorial with some info about geotransforms. (#3132)
OSGeo/gdal: gdal/doc/source/tutorials/geotransforms_tut.rst

.. _geotransforms_tut:

================================================================================
Geotransform Tutorial
================================================================================

Introduction to Geotransforms:
-------------------------------

> 

Reviewer: A geotransform is an affine transformation from the image coordinate space (row, column) to the georeferenced coordinate space (projected or geographic coordinates). It consists in a set of 6 coefficients:

Author: added

0 = x-coordinate of the upper-left corner of the upper-left pixel
1 = w-e pixel resolution / pixel width,
2 = row rotation (typically zero)

> 3 = y-coordinate of the of the upper-left corner of the upper-left pixel

Reviewer: ```suggestion
3 = y-coordinate of the upper-left corner of the upper-left pixel
```

4 = column rotation (typically zero)

> 5 = n-s pixel resolution / pixel height (typically negative value)

Reviewer: ```suggestion
5 = n-s pixel resolution / pixel height (negative value for a north-up image)
```

Xgeo = gt(0) + Xpixel * gt(1) + Yline * gt(2)
Ygeo = gt(3) + Xpixel * gt(4) + Yline * gt(5)

In case of north up images:
----------------------------

> (GT(2), GT(4)) coefficients are zero,

Reviewer: inconsistence case of gt in previous paragraph and GT here

Author: fixed

(GT(1), GT(5)) is pixel size
(GT(0), GT(3)) position is the top left corner of the top left pixel of the raster.

> Note that the pixel/line coordinates in the above are from (0.0,0.0) at the top left corner of the top left pixel

Reviewer: those 2 sentences apply to the general case and should be moved to the previous paragraph

Author: done

.. _geotransforms_tut:

================================================================================
Geotransform Tutorial
================================================================================

Introduction to Geotransforms:
-------------------------------
A geotransform is an affine transformation from the image coordinate space (row, column),
also known as (pixel, line)
to the georeferenced coordinate space (projected or geographic coordinates).

A geotransform consists in a set of 6 coefficients:
----------------------------------------------------

| ``GT(0)`` x-coordinate of the upper-left corner of the upper-left pixel.
| ``GT(1)`` w-e pixel resolution / pixel width.
| ``GT(2)`` row rotation (typically zero).
| ``GT(3)`` y-coordinate of the upper-left corner of the upper-left pixel.
| ``GT(4)`` column rotation (typically zero).
| ``GT(5)`` n-s pixel resolution / pixel height (negative value for a north-up image).

Transformation from image coordinate space to georeferenced coordinate space:
-------------------------------------------------------------------------------

> | ``X_geo = GT(0) + X_pixel | GT(1) + Y_line | GT(2)``

Reviewer: Haven't tested how it looks in .rst but the | character isn't appropriate for multiplication
```suggestion
| ``X_geo = GT(0) + X_pixel * GT(1) + Y_line * GT(2)``
```
Same for below formula

Author: Thanks, that's a result of an incautious search replace...

.. _geotransforms_tut:

================================================================================
Geotransform Tutorial
================================================================================

Introduction to Geotransforms:
-------------------------------
A geotransform is an affine transformation from the image coordinate space (row, column),
also known as (pixel, line)
to the georeferenced coordinate space (projected or geographic coordinates).

A geotransform consists in a set of 6 coefficients:

> ----------------------------------------------------

Reviewer: This line with dashes should probably be removed as it appears as a title line and just replaced by a line with a line-feed. Please check that the rendering is OK

```suggestion

```
